File: DemoServer/Demos/Advanced/StreamingApi.cs
using DemoServer.Controllers;
using DemoServer.Entities;
using DemoServer.Helpers;
using Microsoft.AspNetCore.Mvc;

namespace DemoServer.Demos.Advanced
{
    public partial class AdvancedController : BaseController
    {
        [HttpGet]
        [Route("/advanced/streamingApi")]
        [Demo("Streaming Api", DemoOutputType.Flatten, demoOrder: 210)]
        public object StreamingApi()
        {
            using (var session = DocumentStoreHolder.Store.OpenSession())
            {
                var query = session.Query<Order>("OrderByCompanyAndCountry");

                int count = 0;

                using (var e = session.Advanced.Stream(query))
                {
                    while (e.MoveNext())
                    {
                        // do something with this
                        Order order = e.Current.Document;
                        count++;
                    }
                }

                return new { count };
            }
        }
    }
}

File: DemoServer/PythonDemo/demo_methods/Advanced/StreamingApi.py
from demo.holder import DocumentStoreHolder
from demo.entities import Order

class StreamingApi(object):
	@staticmethod
	def streaming_api:
		with DocumentStoreHolder.get_store().open_session() as session:
            query = session.query(object_type=Order, index_name="OrderByCompanyAndCountry")
			count = 0
            results = session.advanced.stream(query)
            for result in results:
				# do something with this
				order = result.get("document", None)
                count += 1